src/Language/Haskell/Liquid/Bare/Check.hs

```diff
@@ -192,11 +199,14 @@ checkTy :: (Doc -> Error) -> TCEmb TyCon -> TCEnv -> SEnv SortedReft -> Located
 checkTy mkE emb tcEnv env t = mkE <$> checkRType emb env (val $ txRefSort tcEnv emb t)
 
 checkDupIntersect     :: [(Var, Located SpecType)] -> [(Var, Located SpecType)] -> [Error]
-checkDupIntersect xts mxts = concatMap mkWrn dups
+checkDupIntersect xts asmSigs = concatMap mkWrn $ trace msg dups
```

Comment:
Yes my bad sorry!

> On Apr 18, 2016, at 3:27 PM, Eric Seidel <notifications@github.com> wrote:
> 
> In src/Language/Haskell/Liquid/Bare/Check.hs:
> 
> > @@ -192,11 +199,14 @@ checkTy :: (Doc -> Error) -> TCEmb TyCon -> TCEnv -> SEnv SortedReft -> Located
> >  checkTy mkE emb tcEnv env t = mkE <$> checkRType emb env (val $ txRefSort tcEnv emb t)
> >  
> >  checkDupIntersect     :: [(Var, Located SpecType)] -> [(Var, Located SpecType)] -> [Error]
> > -checkDupIntersect xts mxts = concatMap mkWrn dups
> > +checkDupIntersect xts asmSigs = concatMap mkWrn $ trace msg dups
> please! :)
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>